Automattic/jetpack
# At a Glance: add list of activity log events to Activity card

This PR fetches the list of activity log events (up to 20), and displays them in the Activity card in the At a Glance section in the Jetpack dashboard.

Note to designers: here's a quick layout of the items, this is were this PR needs design work.

<img width="495" alt="Captura de Pantalla 2019-05-24 a la(s) 21 21 30" src="https://user-images.githubusercontent.com/1041600/58362246-8507eb80-7e6b-11e9-96cf-a68de93b2be3.png">

#### Changes proposed in this Pull Request:

* add JS to call the endpoint `jetpack/v4/site/activity` and add the response to the app state
* add component `QuerySiteActivity` to fetch the data
* display list of activity log events on the Activity card
* update text at the bottom of the card from `View site activity` to `View all site activity`

#### Testing instructions:






* Test with a connected Jetpack site with Activity Log
* Go to Jetpack Dashboard
* The Activity card should display the event list (if the card itself is not showing, it's not because of this PR since this doesn't change that logic. Maybe you have some issue on your site like Rewind inactive. In such case, use the Dev Tools to mock Rewind to be active)

#### Proposed changelog entry for your changes:

* Users can now see the activity log of their site–up to 20 events–in the Jetpack dashboard.
